CodeOfConduct.md

```diff
@@ -4,7 +4,7 @@
 
 All members and visitors of Hobart Makers Inc. agree to this code of conduct. It helps Hobart Makers Inc. maintain a safe and inclusive environment for all.
 
-Information on how to report incidents that may be in breach of this code of conduct ... ** TODO **
+Please report incidents that may be in breach of this code of conduct to the management committee in person, by email committee@hobartmakers.com, or any other convenient method.
```

Comment:
yes at the conference we had "safety@" which I really like that email address. gave everyone the warm and fuzzies